_posts/2019-11-20-gProfiler.md

```diff
@@ -0,0 +1,36 @@
+---
+site: freiburg
+tags: [tools]
+title: gProfiler is published and running on usegalaxy.eu
+---
+
+[gProfiler](https://biit.cs.ut.ee/gprofiler) is a toolset for functional enrichment analysis and conversions of gene lists.
+
+Biological data analysis often deals with lists of genes arising from various studies. The gProfiler toolset is widely used for finding biological categories enriched in gene lists, conversions between gene identifiers and mappings to their orthologs. The mission of gProfiler is to provide a reliable service based on up-to-date high quality data in a convenient manner across many evidence types, identifier spaces and organisms.
+
+gProfiler relies on data retrieved from Ensembl database and fungi, plants or metazoa specific versions of Ensembl Genomes, and parasite specific data from WormBase ParaSite. In addition to Gene Ontology, supported data sources include KEGG, Reactome, WikiPathways, TRANSFAC, miRTarBase, Human Protein Atlas, CORUM and HP.
+
+gProfiler supports close to [500 organisms](https://biit.cs.ut.ee/gprofiler/page/organism-list) and accepts [hundreds of identifier types](https://biit.cs.ut.ee/gprofiler/page/namespaces-list).
+
+gProfiler has been supporting developers by offering a standardized access through public APIs. The usage of APIs was simplified by wrapping them into Python and R packages. The latter, [gprofiler2](https://CRAN.R-project.org/package=gprofiler2) (which was published as a Conda Forge package named [r-gprofiler2](https://anaconda.org/conda-forge/r-gprofiler2)), was wrapped to be used as a set of Galaxy tools:
+
+- [gProfiler GOSt](https://toolshed.g2.bx.psu.edu/view/iuc/gprofiler_gost/) performs functional enrichment analysis, also known as over-representation analysis (ORA) or gene set enrichment analysis, on input gene list; in addition to the detailed tabular results, it can also create a Plotly graph with the Manhattan plot very similar to the one shown by the [web tool](https://biit.cs.ut.ee/gprofiler/).
+- [gProfiler Convert](https://toolshed.g2.bx.psu.edu/view/iuc/gprofiler_convert/) enables to convert between various gene, protein, microarray probe and numerous other types of namespaces
```

Comment:
maybe you want to link here the real galaxy tools ... so the tool opens right away?

Reply:
Great idea! I'll fix it.